source/agora/flash/Node.d

```diff
@@ -469,14 +469,21 @@ public class FlashNode : FlashControlAPI
     /// Handle an outgoing gossip event
     private void handleGossip (GossipEvent event)
     {
-        foreach (peer; this.known_peers.byKeyValue)
+        scope should_gossip = (in ChannelConfig conf, in PublicKey key)
```

Comment:
I feel like I haven't thought this through. We need to gossip the channel so that a private channel can receive multi hop payments. 🤦 Will revert and add a test